src/crimson/osd/pg.cc

```diff
@@ -12,7 +12,7 @@ PG::PG(spg_t pgid,
        std::string&& name,
        ec_profile_t&& ec_profile,
        cached_map_t osdmap,
-       ceph::net::Messenger* msgr)
+       ceph::net::Messenger& msgr)
```

Comment:
this change should have been in bcc64d2f23978e410f51115e010f8de56db66777 . but it was lost while performing rebasing.